Sources/JavaScriptKit/FundamentalObjects/JSClosure.swift

```diff
@@ -72,33 +40,29 @@ public class JSClosure: JSObject, JSClosureProtocol {
     }
 
     public init(_ body: @escaping ([JSValue]) -> JSValue) {
-        // 1. Fill `id` as zero at first to access `self` to get `ObjectIdentifier`.
-        super.init(id: 0)
-        let objectId = ObjectIdentifier(self)
-        let funcRef = JavaScriptHostFuncRef(bitPattern: Int32(objectId.hashValue))
-        // 2. Retain the given body in static storage by `funcRef`.
-        sharedClosures[funcRef] = body
-        // 3. Create a new JavaScript function which calls the given Swift function.
+        self.hostFuncRef = closureRef
+        closureRef += 1
+
+        // Retain the given body in static storage by `closureRef`.
+        sharedClosures[self.hostFuncRef] = body
+
+        // Create a new JavaScript function which calls the given Swift function.
         var objectRef: JavaScriptObjectRef = 0
-        _create_function(funcRef, &objectRef)
+        _create_function(self.hostFuncRef, &objectRef)
 
-        hostFuncRef = funcRef
-        id = objectRef
+        super.init(id: objectRef)
     }
 
-    public func release() {
-        isReleased = true
-        sharedClosures[hostFuncRef] = nil
-    }
+    @available(*, deprecated, message: "JSClosure.release() is no longer necessary")
+    public func release() {}
```

Comment:
So `JSClosure` in browsers without `FinalizationRegistry` support won't be deallocated, right?

Reply:
If `FinalizationRegistry` is not supported, the JS side will crash on startup when it attempts to call `new FinalizationRegistry()`.